modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/entries/scheduler/test/CheckObjectEntrySchedulerJobConfigurationTest.java

```diff
@@ -289,20 +290,22 @@ public void testCheckObjectEntryVersionsWithMaximumRetentionPeriod()
 		_updateLatestObjectEntryVersion(_getPastDate(3), objectEntry);
 
 		objectEntry = _objectEntryLocalService.updateObjectEntry(
-			TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
+			objectEntry.getObjectEntryId(), TestPropsValues.getUserId(),
+			ObjectEntryFolderConstants.PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+			ServiceContextTestUtil.getServiceContext(),
```

Comment:
Can we replace `ObjectEntryFolderConstants.PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT` with `objectEntry.getObjectEntryFolderId()`, please?